src/components/svg_image.rs

```diff
@@ -97,3 +97,21 @@ pub fn counterclockwise_img() -> Html {
         </svg>
     }
 }
+
+#[function_component(UnsubscribedImg)]
+pub fn search_img() -> Html {
+    html! {
+        <svg aria-label="Отписка" xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-bell-slash-fill" viewBox="0 0 16 16">
+            <path d="M5.164 14H15c-1.5-1-2-5.902-2-7 0-.264-.02-.523-.06-.776zm6.288-10.617A4.988 4.988 0 0 0 8.995 2.1a1 1 0 1 0-1.99 0A5.002 5.002 0 0 0 3 7c0 .898-.335 4.342-1.278 6.113l9.73-9.73M10 15a2 2 0 1 1-4 0zm-9.375.625a.53.53 0 0 0 .75.75l14.75-14.75a.53.53 0 0 0-.75-.75z"/>
+        </svg>
+    }
+}
+
+#[function_component(SubscribedImg)]
+pub fn search_img() -> Html {
```

Comment:
```suggestion
pub fn subscribed_img() -> Html {
```

Reply:
+ we using generic names for icons, for example `bell`

Reply:
done